front/src/components/Affaires/BalanceFromFile/BalanceFromFile.vue

```diff
@@ -230,7 +230,7 @@ export default {
             response_data.push(...response_i.data)
           })
 
-          let relation = this.initRelationArray(response_data);
```

Comment:
I do not get why this was suppressed...

Reply:
This function `initRelationArray` was used to format the server response. But the webservice has been changed in order to serve directly the response in a correct format. This function is now useless and can be removed.